core/workloads/containerized_workload/containerized_workload_schema.json

```diff
@@ -0,0 +1,361 @@
+{
```

Comment:
Cross-linking https://github.com/oam-dev/spec/pull/319/files#r384207097. How would we feel about standardising on YAML OpenAPI v3 schemas instead of JSON schemas?

Reply:
Let's discuss this and the k8s link problem in another PR.

Reply:
I agree we should use YAML OpenAPI v3 schemas.

More than that, this schema is out of date, as it follows the old version of OAM spec.